examples/concurrent-transfer.eg.ts

```diff
@@ -0,0 +1,139 @@
+import { AccountUpdate, Mina, PrivateKey, PublicKey, TokenId, UInt64 } from "o1js"
+import { FungibleToken } from "../index.js"
+
+const url = "https://proxy.devnet.minaexplorer.com/graphql"
+const fee = 1e8
+
+type KeyPair = { publicKey: PublicKey; privateKey: PrivateKey }
+
+const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms))
+
+async function getInferredNonce(publicKey: string) {
+  const query = `
+query {
+  account(publicKey: "${publicKey}") {
+    inferredNonce
+  }
+}`
+
+  const response = await fetch(
+    url,
+    {
+      method: "POST",
+      body: JSON.stringify({ operationName: null, query, variables: {} }),
+      headers: { "Content-Type": "application/json" },
+    },
+  )
+  const json = await response.json()
+  return Number(json.data.account.inferredNonce)
+}
+
+async function sendNoWait(
+  feepayer: KeyPair,
+  from: KeyPair,
+  to: PublicKey,
+  amount: number,
+  payCreationFee: boolean,
+) {
+  const nonce = await getInferredNonce(feepayer.publicKey.toBase58())
+  console.log("feepayer nonce:", nonce)
+  const transferTx = await Mina.transaction({
+    sender: feepayer.publicKey,
+    fee,
+    nonce,
+  }, async () => {
+    if (payCreationFee) {
+      AccountUpdate.fundNewAccount(feepayer.publicKey, 1)
+    }
+    await token.transfer(from.publicKey, to, new UInt64(amount))
+  })
+  await transferTx.prove()
+
+  transferTx.sign([from.privateKey, feepayer.privateKey])
+  const result = await transferTx.send()
+  console.log("Transfer tx:", result.hash)
+
+  // 3 sec for node to update nonce
+  await sleep(3000)
+}
+
+Mina.setActiveInstance(Mina.Network(url))
+
+const _ = PrivateKey.fromBase58("EKE5nJtRFYVWqrCfdpqJqKKdt2Sskf5Co2q8CWJKEGSg71ZXzES7")
```

Comment:
Can we give this a name? The underscore tells the TS compiler to ignore a variable unused error or warning (depending on how you've configured TS).

Reply:
done